torchci/components/JobAnnotationToggle.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { JobData } from "../lib/types";
+import { ToggleButtonGroup, ToggleButton } from "@mui/material";
+
+type JobAnnotation =
+  | "infra_flake"
+  | "time_out"
+  | "SEV"
+  | "broken_trunk"
+  | "test_flake"
+  | "null";
```

Comment:
Seems like this could be an enum? Then the ToggleButtonGroup below could be turned into a for-each loop so that when we add a new category here it'll automatically generate the html for it